test/e2e/config/vsphere.yaml

```diff
@@ -217,7 +217,7 @@ providers:
     versions:
       - name: v1.8.6-0-gde75746a
         # Use manifest from source files
-        value: "https://storage.googleapis.com/artifacts.k8s-staging-capi-vsphere.appspot.com/vm-operator/v1.8.6-0-gde75746a.yaml"
+        value: "file://../../../../cluster-api-provider-vsphere/test/infrastructure/vm-operator/vm-operator-v1.8.6-0-gde75746a.yaml"
```

Comment:
Not sure why we are going so far up that we have to go back into the CAPV repo dir, but I copied this from vc-sim.

One reason could be that this config file can even be used from 4 levels deep in another repo (?)

Reply:
I think this is because paths are relative to the test file.
if this is true, probably we are going up too much, but this seems a generalized smell all around this file

Reply:
Yup, agree :)